front/app/modules/commercial/analytics/admin/utils/query.ts

```diff
@@ -25,9 +25,12 @@ type DateFilter = {
 
 export const getDateFilter = (
   filter: string,
-  startAt: string | null | undefined,
-  endAt: string | null | undefined
+  startAtMoment: Moment | null | undefined,
+  endAtMoment: Moment | null | undefined
```

Comment:
Changed this function to accept moments instead of strings, otherwise we have to covert them to iso strings every time in the `query` function

Reply:
I agree, but unfortunately at this moment the export menu is still used in other places where `startAt` and `endAt` are strings, so there it's more convenient if the component is expecting strings- otherwise we would have to convert it to `Moment`s there first just for the export menu. In our new components `startAt` and `endAt` are kept as `Moment`s for as long as possible so that we can still perform date math on them without needing to convert them. Once the majority of our application works like that I think we should definitely change the export menu to expect `Moment`s!